x/token/handler.go

```diff
@@ -393,3 +399,72 @@ func handleMsgTokenModify(ctx sdk.Context, keeper Keeper, msg types.MsgTokenModi
 	)
 	return sdk.Result{Events: ctx.EventManager().Events()}
 }
+
+func handleMsgTokenActive(ctx sdk.Context, keeper Keeper, msg types.MsgTokenActive, logger log.Logger) sdk.Result {
+	certifiedToken := keeper.GetCertifiedToken(ctx, msg.ProposalID)
```

Comment:
The token info is stored in proposal, so I think handleMsgTokenActive can get info from proposal by proposal ID to activate the certified token.

Reply:
Still keep this way, but add delete operation
When the certified token is activated, delete the certified token data